src/tss2-mu/tpml-types.c

```diff
@@ -113,7 +113,6 @@ TSS2_RC Tss2_MU_##type##_Unmarshal(uint8_t const buffer[], size_t buffer_size, \
     size_t  local_offset = 0; \
     UINT32 i, count = 0; \
     TSS2_RC ret = TSS2_RC_SUCCESS; \
-    type local_dst; \
```

Comment:
This is still needed. The Marshaling spec says that is dest is NULL and offset is not NULL the unmarshaling function needs to set the offset to size that would be needed to unmarshal given type. In this case it needs the local_dst var to unmarshal it and to obtain the size.

Reply:
Yes, and that still happens... We merely call the subfield functions with dest=NULL as well...